monitoring/ceph-mixin/dashboards_out/cephfs-overview.json

```diff
@@ -334,7 +334,6 @@
       "refresh_intervals": [
          "5s",
          "10s",
-         "15s",
```

Comment:
This is acceptable, I guess, right?

Reply:
Well actually node-exporter refresh every 30 seconds too, so I put the same delay everywhere... It would be two times the default scrape interval.